python/tank/pipelineconfig_factory.py

```diff
@@ -279,29 +278,38 @@ def _get_configuration_context():
         # Once found, we can double check that the current Entity is actually
         # associated with the project that the PC is associated with.
         val = pipelineconfig_utils.get_config_install_location(curr_pc_path)
-        
+
     return val
 
 
+def _cleanup_separators(path):
+    """
+    Flips all slashes to the operating system's standard slash separator and strips
```

Comment:
I guess it's personal. I think the code is cleaner this way. It's clearer where the cleaning up of the path starts and ends. Plus it's reusable now! :)

By the way, using os.path.normpath will strip extra slashes at the end of the path, unless that path is C:\. Your code won't respect that.